js/constants/constants.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
     ERROR_SCREEN: "errorScreen"
   },
 
+  CUSTOM_EVENTS: {
```

Comment:
Are you using Custom_Events to hide events from being public? I think the custom event being named the same thing as the regular event may be confusing.  Should we name it DEFFERED_INITIAL_PLAY_REQUESTED?